### PR TITLE
Command buffer test adjustments

### PR DESCRIPTION
**Pull request by gorazd-sumkovski-arm**

### CODE REVIEW

File: test_conformance/extensions/cl_khr_command_buffer/basic_command_buffer.cpp [hunk 1]
```diff
@@ -177,6 +180,71 @@ cl_int BasicCommandBufferTest::SetUp(int elements)
...
+    using BasicCommandBufferTest::BasicCommandBufferTest;
+
+    cl_int Run() override
+    {
+        cl_command_buffer_properties_khr flags = 0;
+        size_t num_flags_set = 0;
+        bool mutli_flags_supported = true;
+        cl_int error = CL_SUCCESS;
+
+        // First try to find mutliple flags that are supported by the driver and
```
Comment (EwanC) [inline test_conformance/extensions/cl_khr_command_buffer/basic_command_buffer.cpp:196]:
```suggestion
        // First try to find multiple flags that are supported by the driver and
```

File: test_conformance/extensions/cl_khr_command_buffer/basic_command_buffer.cpp [hunk 2]
```diff
@@ -177,6 +180,71 @@ cl_int BasicCommandBufferTest::SetUp(int elements)
...
+        }
+
+        if (is_extension_available(
+                device, CL_KHR_COMMAND_BUFFER_MUTABLE_DISPATCH_EXTENSION_NAME))
+        {
+            flags |= CL_COMMAND_BUFFER_MUTABLE_KHR;
+            num_flags_set++;
+        }
+
+        // If we can't find mutliple supported flags, still set a bitfield but
```
Comment (EwanC) [inline test_conformance/extensions/cl_khr_command_buffer/basic_command_buffer.cpp:217]:
```suggestion
        // If we can't find multiple supported flags, still set a bitfield but
```

File: test_conformance/extensions/cl_khr_command_buffer/basic_command_buffer.cpp [hunk 3]
```diff
@@ -177,6 +180,71 @@ cl_int BasicCommandBufferTest::SetUp(int elements)
...
+// multiple flags are set.
+struct MultiFlagCreationTest : public BasicCommandBufferTest
+{
+    using BasicCommandBufferTest::BasicCommandBufferTest;
+
+    cl_int Run() override
+    {
+        cl_command_buffer_properties_khr flags = 0;
+        size_t num_flags_set = 0;
+        bool mutli_flags_supported = true;
```
Comment (EwanC) [inline test_conformance/extensions/cl_khr_command_buffer/basic_command_buffer.cpp:193]:
```suggestion
        bool multi_flags_supported = true;
```

**Review by EwanC — DISMISSED** [review]
LGTM apart from some trivial typos to fix

**Review by EwanC — APPROVED** [review]

**Review by bashbaug — APPROVED** [review]

**Comment by bashbaug** [comment]
Merging as discussed in the November 26th teleconference.

**Pull request merged**